code/modules/antagonists/heretic/influences.dm

```diff
@@ -98,7 +98,7 @@
 	tracked_heretics |= heretic
 
 	// If our heretic's on station, generate some new influences
-	if(ishuman(heretic.current) && !is_centcom_level(heretic.current.z))
+	if(ishuman(heretic.current)) //&& !is_centcom_level(heretic.current.z)) //SKYRAT EDIT
```

Comment:
instead do an inverted check to make sure they are in the interlink area.

Reply:
Done